accounts/forms.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib.auth.forms import UserCreationForm
+from accounts.models import Account
+
+
+class SignUpForm(UserCreationForm):
```

Comment:
I don't think we need a form as we are creating an API

Reply:
noted sir, it was just for checking the model's working